### ISSUE TITLE
Indentation in ns form different from emacs

### ISSUE THREAD

**Issue by den1k** [cursive-ide/cursive#1621]
```
(ns my-ns.indented-in-cursive
  (:require 
    [clojure.pprint]))

(ns my-ns.indented-in-emacs
  (:require
   [clojure.pprint]))
```

**Comment by cursive-ide** [comment]
Yes, in the next version of Cursive I'm going to rewrite the formatter to be (by default, at least) completely compatible with cljfmt, and to use its configuration style. So this should be fixable then, assuming cljfmt can also use the emacs style.

**Comment by holyjak** [comment]
Awesome! Do you have any estimate of when that might be done?

**Comment by cursive-ide** [comment]
Not really, but hopefully on the timescale of a month or so for the EAP. cljfmt is relatively simple but I'll have to test it on a lot of code to ensure complete compatibility.

**Comment by scottlowe** [comment]
💥 Bump! 💥

This is still a daily issue within our organisation.

It wastes more of my time to go back and manually reformat requires than it does to automatically include references whilst I'm coding elsewhere in a namespace. 🙁 

Any chance this issue could be given a higher priority?

**Comment by holyjak** [comment]
Right, but that applies to all forms so it is very heavy handed.
